### DISCUSSION TITLE
Streaming mode feature

### DISCUSSION THREAD

**Discussion by Kilimanjarogg** [maximmax42/Discord-CustomRP#60] [Ideas]
New feature suggestion, at the moment CustomRP displays only Playing.. any chance to add new future Streaming... e.g in photo https://i1.wp.com/thatoneguy.com.au/oadsooju/2021/09/image.png

It work when people streams on on twitch but not youtube or facebook, so it woulb good solution to bypass it and display that person streaming on youtube or fb etc etc.. usability are endless.

Cheers

**Comment by maximmax42** [comment]
Not possible sadly, only bot accounts can freely choose the activity type for their status.

**Reply by Kilimanjarogg** [reply]
what you mean by saying bot accounts? let say if I start to stream on twitch, discord auto detect it and display that am streaming on twitch and gets all info.. I thought it its kinda similar how now CustomRp works...

BTW, CRP are perfect, hope there is any chance to implement that streaming future, atleast to glich discord and it thinks someone streaming and gets that streaming under the name with purple dot on avatar.

Cheers

**Reply by maximmax42** [reply]
I mean actual bot accounts, with the BOT badge next to the username. They can set whatever status they want: listening, watching, streaming, playing. User accounts can only have listening status when using Spotify, and streaming when streaming to Twitch/YouTube, and that's it. Games and programs like mine can only have playing. At least for now, maybe some time later they will allow that for anyone.

**Reply by Kilimanjarogg** [reply]
okay, but my acc not bot, and when I stream to twtich my acc get status streaming if I enable it on discord, kinda confused :/

**Reply by maximmax42** [reply]
Yeah, and that's the only way you can get that status. No other programs can set it, I assume because it needs some sort of approval from Discord.

**Reply by MayhemBill** [reply]
This isn't actually true, you can do this through a certain self bot I didn't use in the past, so it's possible on a user account.

**Reply by maximmax42** [reply]
Didn't know that. Still, my app and others like it can only use "Playing".

**Reply by mosamir99** [reply]
Is there by any chance a new method where this could happen? Can we change the activity type yet?

**Reply by maximmax42** [reply]
Only through self-bot or an RPC plugin for Vencord.
